---
geometry_msgs/PoseStamped[] objects_pose
geometry_msgs/PoseStamped plane_pose